aptos-move/e2e-testsuite/src/tests/transaction_builder.rs

```diff
@@ -48,11 +47,7 @@ fn test_rotate_authentication_key_with_nonce_admin() {
 
         let account = test_env.dr_account;
         let txn = account
-            .transaction()
-            .write_set(WriteSetPayload::Script {
-                script: encode_rotate_authentication_key_with_nonce_admin_script(0, new_key_hash.clone()),
-                execute_as: *new_account.address(),
```

Comment:
hi @zekun000  previous script allow passing an **_execute_as_**. however, no sure how we could pass execute_as for the script function. This tests failed.